releases/release-1.14/exception.yaml

```diff
@@ -9,4 +9,76 @@
   thread: 
   pull_requests:
   - 
-  status: 
+  status:
+  
+  - name: Support Out-of-Tree vSphere Cloud Provider 
+  issue: 670
+  date_requested: Jan 30th, 2019
+  date_reviewed: Jan 30th, 2019
+  thread: https://groups.google.com/forum/#!topic/kubernetes-milestone-burndown/psJ0VI-gt5U
+  pull_requests:
+  - 
+  status: Approved 
+  
+  - name: Kubelet Resource Metrics Endpoint
+  issue: 727
+  date_requested: Jan 30th, 2019
+  date_reviewed: Jan 31st, 2019
+  thread: https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/kubernetes-milestone-burndown/pFGpJkGlOZU/YEjkMngtFAAJ
```

Comment:
Link to the topic instead e.g., `https://groups.google.com/forum/#!topic/kubernetes-milestone-burndown/blahblahblah`